pymeasure/instruments/hp/hp34401A.py

```diff
@@ -74,7 +74,7 @@ def __init__(self, adapter, name="HP 34401A", **kwargs):
         super().__init__(
             adapter,
             name,
-            asrl={'baud_rate': 9600, 'data_bits': 7, 'parity': 2},
+            asrl={'baud_rate': 9600, 'data_bits': 8, 'parity': 0},
```

Comment:
According to [hp34401manual.pdf](https://engineering.purdue.edu/~aae520/hp34401manual.pdf) these are the default settings. Actually it should be 2 stop bits but using pyvisa one should use the enum provided.

Reply:
you could specify the stop bits with `stop_bits=20` (afaik 20 is the integer value of the stop bits 2 enum in pyvisa).

Reply:
Added the stop bits plus a link to where this magic "20" number comes from.